docs/edex/new-grid.md

```diff
@@ -464,10 +316,24 @@ You will have to restart ingestGrib for the changes to take place:
 
       sudo service edex_camel restart ingestGrib
       
-Now you can try [re-ingesting the grib2 file](#grib2-products).
+Now you can try [re-ingesting the grib2 file](#download-test-data).
       
 ---
 
+## Creating Menu Items
+
+After you have confirmed that the grid was ingested with the given name, you can [edit the D2D product menus to display the new grid](../cave/d2d-edit-menus.md).
+
+---
+
+## Using wgrib2
+
+Mentioned in this page are a few command parameters for `wgrib2` such as `-center`, `-subcenter`, and `-full_name`.
```

Comment:
add -grid since that was used as well.

Reply:
also added `-varX` because we reference that one too.